packages/flow-designer/src/components/node/AbstractNode.test.tsx

```diff
@@ -47,6 +47,25 @@ describe('Testing <AbstractNode>', () => {
 		expect(onClick).toHaveBeenCalledTimes(1);
 	});
 
+	it('call the injected onDblClick action when double clicked', () => {
+		const onClick = jest.fn();
+		const onDblClick = jest.fn();
+		const wrapper = mount(
```

Comment:
As part of migration to React 18, you should use react-testing-library for new tests instead of Enzyme